orc8r/cloud/go/obsidian/tests/unit_test_utils.go

```diff
@@ -91,7 +91,7 @@ func RunUnitTest(t *testing.T, e *echo.Echo, test Test) {
 			expectedBytes, err := test.ExpectedResult.MarshalBinary()
 			if assert.NoError(t, err) {
 				// Convert to string for more readable assert failure messages
-				assert.Equal(t, string(expectedBytes), string(recorder.Body.Bytes()))
+				assert.Equal(t, string(expectedBytes)+"\n", string(recorder.Body.Bytes()))
```

Comment:
Not entirely sure which package update caused this change to be necessary, but it seems like json encoding began adding a newline.